### ISSUE TITLE
Optimize podcast section

### ISSUE THREAD

**Issue by manujuniris** [unbloxtech/unblox-website#9]
- [x] modify the dimensions of image in podcast section
- [x] add a share button for 3tb social platforms
- [x] explore soultion for automating podcast using rss feed or api

**Comment by manujuniris** [comment]
Hi @nilesh-uniris and @samiran-ghosh

1. Dimensions of image in podcast section has been modified.
https://unbloxtech.github.io/pre-production/#podcasts
3. Added a share button for 3tb social platforms (Linkedin and Facebook), we can also add Twitter once twitter handle is created. 
https://unbloxtech.github.io/pre-production/#podcasts
5. Podcast page has been automated, a new release of podcast will directly be reflected on - https://unbloxtech.github.io/pre-production/podcasts.html
To automate it, I have used an api from omny.fm which also leverage us to showcase publication date along with podcast player - https://api.omny.fm/orgs/4bb33704-615b-4054-aae9-ace500fd4197/programs/daeea5e9-9f88-4f9e-b527-adfa0088c647/clips
